watchtower/wtdb/client_db.go

```diff
@@ -1799,18 +1799,24 @@ func isSessionClosable(sessionsBkt, chanDetailsBkt, chanIDIndexBkt kvdb.RBucket,
 	}
 
 	commitsBkt := sessBkt.NestedReadBucket(cSessionCommits)
-	if commitsBkt == nil {
-		// If the session has no cSessionCommits bucket then we can be
-		// sure that no updates have ever been committed to the session
-		// and so it is not yet exhausted.
-		return false, nil
+	if commitsBkt != nil {
```

Comment:
Isn't this now doing the exact same thing as we already do below? But now we'd run into a nil pointer panic if the bucket was nil.